script/build.sh

```diff
@@ -53,6 +53,12 @@ cp -r $DIR/NOTICE.md ${SRC_DIR}/performance/
 # Copy the docs into portal
 cp -r $DIR/docs ${SRC_DIR}/pfe/portal/
 
+# Copy the appsody extension into portal. The zip file must have a version number e.g. codewind-appsody-extension-0.3.0.zip
+# in order for it to be accepted as a valid extension
+mkdir -p ${SRC_DIR}/pfe/extensions
+rm -f ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-0.3.0.zip
+curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-0.3.0.zip https://github.com/eclipse/codewind-appsody-extension/archive/0.3.0.zip
```

Comment:
I guess we need to update this file to pick up a newer release (as opposed to grabbing a latest release)

Don't really have have a strong opinion on this but just want to point it out.